src/python/Python-2.7.8/setup.py

```diff
@@ -803,8 +803,8 @@ def detect_modules(self):
                                libraries=math_libs) )
         # Detect SSL support for the socket module (via _ssl)
         search_for_ssl_incs_in = [
-                              '/usr/local/ssl/include',
                               '/usr/sfw/include/',
+                              '/usr/local/ssl/include',
```

Comment:
in this case, I think that it needs an inline comment so that other will know that the order is important